packages/web-components/fast-foundation/src/design-token/design-token.ts

```diff
@@ -330,16 +343,20 @@ class DesignTokenNode<T> {
         return this.target.contains(node.target);
     }
 
-    private findParentNode() {
+    private findParentNode(): DesignTokenNode<T> | null {
+        if (this.target === defaultElement) {
+            return null;
+        }
+
         if (this.target !== document.body && this.target.parentNode) {
             const container = DI.getOrCreateDOMContainer(this.target.parentElement!);
             try {
                 return container.get(DesignTokenNode.channel(this.token));
             } catch (e) {
-                return null;
+                return DesignTokenNode.for(this.token, defaultElement);
```

Comment:
I wonder if we should consider adding a method to the container such as `tryGet` that doesn't throw if the service is not found, but returns null instead.

Reply:
Yea, that's a good call. I'll add that and work it in here.

Reply:
Actually `Container` has a `has()` method, is it preferable to just test prior to `get()` if the author wants to avoid the error?

Reply:
I suppose that would be twice the resolution work; I'll just add it

Reply:
Actually that looks a little more involved than I first thought, I'll take it as a follow up

Reply:
There might not be an easy way to add it at all. I haven't looked in a while. Just a passing thought. If we can find a way, it's a potential improvement for later.

Reply:
I think there is, but `jitRegister` throws as well as `get()` so there are a few code-paths to patch up. https://github.com/microsoft/fast/issues/4582 is tracking and I'll add a TODO in this block